src/sql/base/browser/ui/button/button.ts

```diff
@@ -23,6 +23,18 @@ export class Button extends vsButton {
 			this.$el.style('outline-color', this.buttonFocusOutline ? this.buttonFocusOutline.toString() : null);
 			this.$el.style('outline-width', '1px');
 		});
+
+		this.$el.on(DOM.EventType.MOUSE_DOWN, (e) => {
+			const mouseEvent = e as MouseEvent;
+			if (!this.$el.hasClass('disabled') && mouseEvent.button === 0) {
```

Comment:
Does this have impact outside of the extensibility scenarios? Can this impact any core scenario?

Reply:
It shouldn't. All it does is adding/removing "active" class in the dom. VS button doesn't use active class for anything

---
In reply to: [216546546](https://github.com/Microsoft/sqlopsstudio/pull/2488#discussion_r216546546) [](ancestors = 216546546)